source/Irrlicht/COpenCLHandler.cpp

```diff
@@ -14,6 +14,7 @@ clGetGLContextInfoKHR_fn COpenCLHandler::pClGetGLContextInfoKHR = NULL;
 cl_platform_id COpenCLHandler::platforms[IRR_MAX_OCL_PLATFORMS];
 cl_uint COpenCLHandler::actualPlatformCount = 0;
 COpenCLHandler::SOpenCLPlatformInfo COpenCLHandler::platformInformation[IRR_MAX_OCL_PLATFORMS];
+cl_context COpenCLHandler::openCLContext = 0;
```

Comment:
this shoud not be a static/singleton multiple contexts should be possible

Reply:
whole COpenCLHandler is current used as static, so everything is static
I will store a COpenCLHandler object in COpenGLDriver, is that OK?